test/sanity/issue5730-add-node-in-permissions/main.js

```diff
@@ -0,0 +1,13 @@
+
+$(function() {
+
+   var wv = document.querySelector('webview');
+   document.getElementById("request1").innerHTML = JSON.stringify(wv.request);
+
+   setTimeout(function(){
+      document.getElementById("request2").innerHTML = JSON.stringify(wv.request);
+   }, 500);
```

Comment:
@rogerwang Please see here, the id "request2" will be added content after waiting 500 ms, so we need to wait a minute to load this content, otherwise, we only can find the id without content. I tried to use loop and wait, but it didn't work.

Reply:
sleeping for 1 second does not guaranteed to work, because the process can be stalled due to system activity. It may actually take longer than 1 sec. So we need loop and wait.

Reply:
Updated